pulp_2to3_migration/tests/functional/test_migration_behaviour.py

```diff
@@ -401,3 +401,95 @@ def test_no_immediate_importer(self):
 
         self.assertEqual(self.file_repo_versions_api.list(pulp3_repo.pulp_href).count, 2)
         self.assertEqual(repo_content.count, 3)
+
+    def test_importer_swap(self):
+        """
+        Test that if only importers in migration plan changed, the changes are noticed.
+
+        Importers themselves haven't changed in Pulp 2, only where they are specified in the
```

Comment:
should we add a test when there has been a swap and yet one of the importers/distributors have changed in pulp2?

I know that we have a separate test that ensures importers are re-migrated if feed changes and similar for distributors config i just wonder if it makes sense to combine both cases together. I will come back to this suggestion tomorrow morning ( maybe it's overkill)

Reply:
I think I can add such test if we have already a snapshot with the changes. I think we do.